builtin/logical/database/path_roles.go

```diff
@@ -623,14 +686,23 @@ func (b *databaseBackend) pathStaticRoleCreateUpdate(ctx context.Context, req *l
 		}
 	}
 
-	item.Priority = lvr.Add(role.StaticAccount.RotationPeriod).Unix()
+	_, rotationPeriodChanged := data.Raw["rotation_period"]
+	_, rotationScheduleChanged := data.Raw["rotation_schedule"]
```

Comment:
Curious if there is a difference between these values from `data.Raw` and `rotationPeriodOk` / `rotationScheduleOk` from above?

Reply:
Nope, good catch!